src/features/encoders/avif/client/index.tsx

```diff
@@ -120,20 +125,21 @@ export class Options extends Component<Props, State> {
         };
 
         const newOptions: EncodeOptions = {
-          cqLevel: optionState.lossless ? 0 : maxQuant - optionState.quality,
-          cqAlphaLevel:
+          quality: optionState.lossless ? MAX_QUALITY : optionState.quality,
+          qualityAlpha:
             optionState.lossless || !optionState.separateAlpha
-              ? -1
-              : maxQuant - optionState.alphaQuality,
+              ? -1 // default AVIF alphaLevel
```

Comment:
"alphaLevel" seems like a typo. I think this comment should point out that -1 means "set qualityAlpha to quality".

Reply:
Yep it is, I'm on it